packages/server/src/orders/domain/CreateOrderHandler.ts

```diff
@@ -39,7 +39,7 @@ class CreateOrderHandler implements Handler<CreateOrderRequest, Order> {
       const created = await this.orderModel.create({ ...req, date: new Date() });
       const mail = await this.authFacade.getEmailByUserId(req.userId);
 
-      if (mail != null) {
+      if (mail != null && process.env.NODE_ENV != 'test') {
```

Comment:
```suggestion
      if (mail != null && process.env.NODE_ENV !== 'test') {
```

Nie wiem czy porównanie `mail != null` ma sens, bo `mail` ma typ `string | undefined`. Biorąc pod uwagę, że mamy abstract equality comparison, to zadziała bo tam `null` i `undefined` są traktowane tak samo, ale porównanie z `null`em zmiennej która nigdy nie może być `null`em jest trochę mylące. (wiem że to nie była zmiana w tym PR, ale jak już jesteśmy w tym nawiasie może warto powiedzieć)

Reply:
to nie moje, tak było więc nie wiem :/ ale no mogę zmienic :D